config/database.yml

```diff
@@ -9,10 +9,14 @@ default: &default
 development:
   <<: *default
   database: shf_project_development
+  username: shf
```

Comment:
The changes in this file are ones that just _you_ need, so you shouldn't submit them.

(be sure **not** to add  this to the project `.gitignore` though)